modules/proto/tests/src/smithytranslate/proto3/internals/CompilerRendererSuite.scala

```diff
@@ -0,0 +1,1459 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package smithytranslate.proto3.internals
+
+import munit._
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.validation.ValidatedResultException
+
+class CompilerRendererSuite extends FunSuite {
```

Comment:
this file probably presents the biggest changes : moved to another folder and a lot of tests were modified or added